scripts/start-setupVanillaTweaks

```diff
@@ -12,73 +12,149 @@ set -e -o pipefail
 . "${SCRIPTS:-/}start-utils"
 isDebugging && set -x
 
-out_dir=/data/${LEVEL:-world}/datapacks
+VT_VERSION=""
+DATAPACKS_DIR="/data/${LEVEL:-world}/datapacks"
+RESOURCEPACKS_DIR="/data/resourcepacks"
 
 # Remove old VANILLATWEAKS
-if isTrue "${REMOVE_OLD_VANILLATWEAKS}" && [ -z "${VANILLATWEAKS_FILE}" ]; then
-    if [ -d "$out_dir" ]; then
-    find "$out_dir" -mindepth 1 -maxdepth ${REMOVE_OLD_VANILLATWEAKS_DEPTH:-16} -wholename "${REMOVE_OLD_VANILLATWEAKS_INCLUDE:-*}" -not -wholename "${REMOVE_OLD_VANILLATWEAKS_EXCLUDE:-}" -delete
+if isTrue "${REMOVE_OLD_VANILLATWEAKS}"; then
+  # NOTE: datapacks include crafting tweaks.
+  if [ -d "$DATAPACKS_DIR" ]; then
+    find "$DATAPACKS_DIR" -mindepth 1 -maxdepth ${REMOVE_OLD_VANILLATWEAKS_DEPTH:-16} -wholename "${REMOVE_OLD_VANILLATWEAKS_INCLUDE:-*}" -not -wholename "${REMOVE_OLD_VANILLATWEAKS_EXCLUDE:-}" -delete
+  fi
+  if [ -d "$RESOURCEPACKS_DIR" ]; then
+    find "$RESOURCEPACKS_DIR" -mindepth 1 -maxdepth ${REMOVE_OLD_VANILLATWEAKS_DEPTH:-16} -wholename "${REMOVE_OLD_VANILLATWEAKS_INCLUDE:-*}" -not -wholename "${REMOVE_OLD_VANILLATWEAKS_EXCLUDE:-}" -delete
   fi
 fi
 
-# Example: VANILLATWEAKS_SHARECODE=MGr52E
-# Code generated from the UI website, typically a alphanumeric 6 digit code.
-if [[ "$VANILLATWEAKS_SHARECODE" ]]; then
-  VANILLATWEAKS_FILE=/tmp/vanillatweaksfile.json
-  SHARECODE_LOOKUP_URL="https://vanillatweaks.net/assets/server/sharecode.php?code=${VANILLATWEAKS_SHARECODE}"
-  curl -f $SHARECODE_LOOKUP_URL -o $VANILLATWEAKS_FILE
-  if [ ! -f "$VANILLATWEAKS_FILE" ]; then
-    log "ERROR: Unable to use share code provided to retreive vanillatweaks file"
+# Download and unzip datapacks
+downloadDatapacks(){
+  VT_FILE=$1
+  DATAPACKS=$(jq -jc '.packs // empty' $VT_FILE)
+  if [ ! "$DATAPACKS" ]; then
+    log "ERROR: unable to retrieve datapacks from ${VT_FILE}"
     exit 2
   fi
-fi
+  VT_ZIPDATA_URL="https://vanillatweaks.net/assets/server/zipdatapacks.php"
 
-# Use vanillatweaks file to specify VT and datapacks
-if [[ "$VANILLATWEAKS_FILE" ]]; then
-  if [ ! -f "$VANILLATWEAKS_FILE" ]; then
-    log "ERROR: given VANILLATWEAKS_FILE file does not exist"
+  log "DATAPACKS: ${DATAPACKS}, VT_VERSION: ${VT_VERSION}"
+  log $(curl -X POST -F "packs=${DATAPACKS}" -F "version=${VT_VERSION}" $VT_ZIPDATA_URL)
```

Comment:
Is this log left over from debugging? There's actually a `debug` that only logs when debugging enabled. 

```suggestion
  debug $(curl -X POST -F "packs=${DATAPACKS}" -F "version=${VT_VERSION}" $VT_ZIPDATA_URL)
```